specifyweb/frontend/js_src/lib/components/Forms/BaseResourceView.tsx

```diff
@@ -117,12 +120,16 @@ export function useResourceView<SCHEMA extends AnySchema>({
       ? formsText.newResourceTitle({ tableName: resource.specifyTable.label })
       : resource.specifyTable.label;
   const title =
-    formatted.length > 0
-      ? commonText.colonLine({
+    formatted.length === 0
+      ? formattedTableName
+      : formatted.length > 0 &&
```

Comment:
you already have a check for `formatted.length === 0` above
here, in the else branch, it is guaranteed that `formatted.length > 0` is always true.
isn't it?
then this check is redundant here